spec/soapy_bing/oauth_credentials_spec.rb

```diff
@@ -83,28 +83,37 @@
 
   describe '#access_token' do
     let(:credentials) { { client_id: 'foo', client_secret: 'bar', refresh_token: 'baz' } }
-    let(:response) { double(:response) } # rubocop:disable RSpec/VerifiedDoubles
+    let(:response) do
+      HTTParty::Response.new(
+        OpenStruct.new(options: {}),
+        OpenStruct.new(code: status_code),
+        -> { response_body }
+      )
+    end
```

Comment:
I wanted to use `instance_double`, but it told me that `HTTParty::Response` doesn't implement the method `[]` for some reason, so I couldn't use it.